src/test/java/uk/gov/hmcts/darts/authorisation/util/EmailAddressFromTokenUtilTest.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.hmcts.darts.authorisation.util;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class EmailAddressFromTokenUtilTest {
```

Comment:
We could add some tests to ensure the priroity order is met i.e. if a token has emails, email and preferred_username claims then we use the appropriate one.

Reply:
done